qa/tasks/cephfs/test_snap_schedules.py

```diff
@@ -577,6 +577,357 @@ def test_snap_schedule_all_periods(self):
         self.mount_a.run_shell(['rmdir', test_dir])
 
 
+class TestSnapSchedulesSubvolAndGroupArguments(TestSnapSchedulesHelper):
```

Comment:
@mchangir This change needs to be verified with a v1 subvolume which is not auto-upgraded to v2 on access if a v1 subvolume has snapshots.

Reply:
ack
I have hacked up something to support a `fs subvolume create_version` command which creates a subvolume for a specific version. This command could help is automating integration testing as well instead of tweaking the `.meta` file.